oslabs-beta/React-Labyrinth
# AL/CR/LK - jest testing

## Overview

**Issue Type**


**Description**
Migrated testing from mocha/chai to jest for VSCode environment.

**Ticket Item**
RL-31: Create testing for React Labyrinth Extension

**Steps to Validate Feature**

1. `npm install` for new devDependencies
2. Run `npm run webpack`
3. Run `npx tsc`
4. Delete previous .vscode-test folder
5. Run `npm run test` and see the sample tests passing

**Previous behavior**
We were using mocha/chai for integration testing but kept running into compiling errors.

**Expected behavior**
We migrated to using jest and added functionality to allow jest to enter the vscode environment and access the global object of vscode.

**Additional Context**
ESLint fixed the spacing of certain files, I took out comments that we don't utilize anymore, and added semi-colons in places that are for best practice. Moved testing-related files into designated testing folders.
